openage/convert/main.py

```diff
@@ -47,18 +47,23 @@ def mount_drs(filename, target, ignore_nonexistant=False):
 
         result.joinpath(target).mount(DRS(drspath.open('rb')).root)
 
-    mount_drs("data/graphics.drs", "graphics")
-    mount_drs("data/interfac.drs", "interface")
-
-    mount_drs("data/sounds.drs", "sounds")
-    mount_drs("data/sounds_x1.drs", "sounds")
+    if result['AoK HD'].exists():
+        # Mounts for HD edition version 4.0
+        result['graphics'].mount(srcdir['resources/_common/drs/graphics'])
+        result['interface'].mount(srcdir['resources/_common/drs/interface'])
+        result['sounds'].mount(srcdir['resources/_common/drs/sounds'])
+        result['gamedata'].mount(srcdir['resources/_common/drs/gamedata'])
+        result['terrain'].mount(srcdir['resources/_common/drs/terrain'])
+        result['data'].mount(srcdir['/resources/_common/dat'])
+    else:
+        # Mounts for AoC
+        mount_drs("data/graphics.drs", "graphics")
+        mount_drs("data/interfac.drs", "interface")
 
-    # In the HD edition, gamedata.drs has been merged into gamedata_x1.drs
-    mount_drs("data/gamedata.drs", "gamedata", ignore_nonexistant=True)
-    mount_drs("data/gamedata_x1.drs", "gamedata")
-    mount_drs("data/gamedata_x1_p1.drs", "gamedata")
+        mount_drs("data/sounds.drs", "sounds")
+        mount_drs("data/sounds_x1.drs", "sounds")
 
-    mount_drs("data/terrain.drs", "terrain")
+        mount_drs("data/terrain.drs", "terrain")
```

Comment:
you still need to mount, `gamedata.drs, gamedata_x1.drs, gamedata_x1_p1.drs` to `gamedata`, currently you removed them for AoC.